src/js/component/Headlines.js

```diff
@@ -1,15 +1,18 @@
+import {view} from '../../Views/View'
+import{headlinesNewsData} from './LazyLoading'
+import {settings} from './Variable';
 export class Headlines {
     constructor(settings){
       this.modalHeadlines = document.querySelector(settings.newsModal)
-      this.showheadlinesBtn = document.querySelector(settings.newsHeadlines)
     }
     async dynamicImport() {
-      const module = await import("./LazyLoading.js");
-      module.default()
-        .then(({ articles }) => {
+      return await import(/* webpackPreload: true */ "./LazyLoading.js")
+      .then(({ articles }) => {
           articles.map(item => {
-            this.modalHeadlines.innerHTML += `<a href="#" class="list-group-item"><strong>${item.title} </strong></li>`;
+            this.modalHeadlines.innerHTML += view.headLinesHTML(item);
           });
         });
     }
 }
+const newsHeadings = new Headlines(settings)
```

Comment:
Currently your headlines are loaded after opening of the site. Could you please fix it to load only when modal is shown?